src/components/SuggestionBox/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
+import Body from '../texts/Body'
+import { theme } from '~/constants'
+import { verticalScale, scale } from 'react-native-size-matters/extend'
+import check from '~/assets/images/UsernameRequest/check.svg'
+import { SvgXml } from 'react-native-svg'
+
+const SuggestionBox = (props) => {
+  const {
+    style,
+    children,
+    ...restProps
+  } = props
+
+  const suggestionsStyle = [
+    styles.suggestionsbox,
+    style
+  ]
+  const textStyle = [
+    styles.text,
+    style
+  ]
+
+  return (
+    <View style={suggestionsStyle} {...restProps}>
+      <Body
+        style={textStyle} {...restProps}
+      >
+        {children}
+      </Body>
+      <SvgXml xml={check} style={styles.icon} />
+    </View>
+  )
+}
```

Comment:
Add props `textStyle` and `boxStyle` and change this with:

```
const suggestionsStyles = [
    styles.suggestionsbox,
    boxStyle
  ]

  const textStyles = [
    styles.text,
    textStyle
  ]
```

and then

```
<View style={suggestionsStyles} {...restProps}>
      <Body
        style={textStyles} {...restProps}
      >
```

By doing this you increase the flexibility of the component